docusaurus/constants.js

```diff
@@ -0,0 +1,29 @@
+const folderMapping = {
+  android: 'Android',
+  flutter: 'Flutter',
+  ios: 'iOS',
+  react: 'React',
+  reactnative: 'React Native',
+};
+
+const platformMapping = {
+  android: 'android',
+  flutter: 'flutter-dart',
+  ios: 'ios-swift',
+  react: 'react',
+  reactnative: 'react-native',
+};
+
+const languageMapping = {
```

Comment:
This will be used to map the sdk to its language. Having the language in the query params is necessary in order to show the code blocks in the search results